kernel_tracing/metric_evaluation/test.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright(c) 2022 Intel Corporation. All rights reserved.
+
+# Run with --help for a description of this script
+
+import argparse
+import pytest
+
+
+def test(case):
+    """ This is the test function that is called by pytest """
+    (_, condition, bpftrace_vars) = case
```

Comment:
```suggestion
    (_ignored_something, condition, bpftrace_vars) = case
```
From an "API" perspective it's useful to know what gets discarded.